src/pipeline/postprocess/parseTQLQuery.ts

```diff
@@ -0,0 +1,345 @@
+import { compute } from '../../engine/compute';
+import { getCurrentSchema } from '../../helpers';
+import type { EnrichedBormEntity, EnrichedBormRelation } from '../../types';
+import type { PipelineOperation } from '../pipeline';
+
+const parseMetaData = (str: string) => {
+	const asRegex = /as:([a-zA-Z0-9_\-·]+)/;
+	const justIdRegex = /justId:([a-zA-Z0-9_\-·]+)/;
```

Comment:
I think this three things can be done in reception by storing the linkfield / datafield id only no?

Reply:
Ok maybe not, because we need also to know stuff about the original query.
Byt maybe we can unify them in a single key 🤔